core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -240,10 +279,37 @@ func (l *logPollerWrapper) LatestEvents() ([]evmRelayTypes.OracleRequest, []evmR
 		}
 	}
 
-	l.lggr.Debugw("LatestEvents: done", "nRequestLogs", len(resultsReq), "nResponseLogs", len(resultsResp), "nextBlock", nextBlock, "latest", latest)
+	l.lggr.Debugw("LatestEvents: done", "nRequestLogs", len(resultsReq), "nResponseLogs", len(resultsResp), "startBlock", startBlock, "endBlock", latest)
 	return resultsReq, resultsResp, nil
 }
 
+func (l *logPollerWrapper) FilterPreviouslyDetectedEvents(logs []logpoller.Log, detectedEvents *detectedEvents, filterType string) []logpoller.Log {
```

Comment:
Doesn't have to be a public method (unless you want to add more tests for it specifically - not a bad idea!)